boards/arm/mec172xmodular_assy6930/mec172xmodular_assy6930.dts

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright (c) 2021, Microchip Technology Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+
+#include <microchip/mec172xnsz.dtsi>
+
+/ {
+	model = "Microchip MEC172XMODULAR_ASSY6930 evaluation board";
+	compatible = "microchip,mec172xmodular_assy6930", "microchip,mec172xnsz";
+
+	chosen {
+		zephyr,sram = &sram0;
+		zephyr,flash = &flash0;
+		zephyr,console = &uart1;
+	};
+
+	aliases {
+		led0 = &led4;
+		led1 = &led3;
+		i2c0 = &i2c_smb_0;
+		i2c1 = &i2c_smb_1;
+		i2c7 = &i2c_smb_2;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+		led4: led_0 {
+			/* GPIO241/CMP_VOUT0/PWM0_ALT on schematic,
+			 * LED4 on silkscreen.
+			 */
+			gpios = <&gpio_240_276 1 GPIO_ACTIVE_HIGH>;
+			label = "LED 4";
+		};
+		led3: led_1 {
+			/* GPIO175/CMP_VOUT1/PWM8_ALT on schematic,
+			 * LED5 on silkscreen.
+			 */
+			gpios = <&gpio_140_176 29 GPIO_ACTIVE_HIGH>;
+			label = "LED 5";
+		};
+	};
+};
+
+&cpu0 {
+	status = "okay";
+};
+
+/* Initialize ECIA. Does not initialize child devices */
+&ecia {
+	status = "okay";
+};
```

Comment:
@wu-frances this is missing several eSPI related entries that should be supported  so modular card can be used with host. i.e. girq24, espi0, kbc0, acpi_ec0

Reply:
PR scope changed, comment not applicable